infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.bicep

```diff
@@ -136,7 +136,9 @@ var varAzBackupGeoCodes = {
 }
 
 // If region entered in parLocation and matches a lookup to varAzBackupGeoCodes then insert Azure Backup Private DNS Zone with appropriate geo code inserted alongside zones in parPrivateDnsZones. If not just return parPrivateDnsZones
-var varPrivateDnsZonesMerge = contains(varAzBackupGeoCodes, parLocation) ? union(parPrivateDnsZones, ['privatelink.${varAzBackupGeoCodes[toLower(parLocation)]}.backup.windowsazure.com']) : parPrivateDnsZones
+var varPrivateDnsZonesMerge = contains(varAzBackupGeoCodes, parLocation) ? union(parPrivateDnsZones, [
+  'privatelink.${varAzBackupGeoCodes[toLower(parLocation)]}.backup.windowsazure.com'
+]) : parPrivateDnsZones
```

Comment:
Dont think this should be touched as part of this PR. Think this may be a mistake?